superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterControls/FilterValue.tsx

```diff
@@ -27,20 +27,15 @@ import {
 } from '@superset-ui/core';
 import { areObjectsEqual } from 'src/reduxUtils';
 import { getChartDataRequest } from 'src/chart/chartAction';
-import Loading from 'src/components/Loading';
 import BasicErrorAlert from 'src/components/ErrorMessage/BasicErrorAlert';
 import { FilterProps } from './types';
 import { getFormData } from '../../utils';
 import { useCascadingFilters } from './state';
-
-const StyledLoadingBox = styled.div`
-  position: relative;
-  height: ${({ theme }) => theme.gridUnit * 8}px;
-  margin-bottom: ${({ theme }) => theme.gridUnit * 6}px;
-`;
+import LoadingBox from '../LoadingBox';
 
 const FilterItem = styled.div`
-  padding-bottom: 10px;
+  min-height: ${({ theme }) => theme.gridUnit * 10.5}px;
+  padding-bottom: ${({ theme }) => theme.gridUnit * 2.5}px;
```

Comment:
The min height needs to be 32px + the bottom padding, which in `gridUnits` is 8 + 2.5 = 10.5.

Reply:
I'm hoping/trying to avoid fractional gridUnits if at all possible. I know there are a few kicking around in the codebase, but (a) we should just learn to stick to the grid for the sake of design's sanity, and (b) ththings can get weird if gridUnit is ever moved to, say, 5px, and we find ourselves in fractional-pixel land.

Reply:
Ha - I looked at the designs, and turns out they were actually 8 and not 10! Will update accordingly

Reply:
Same with the line above btw. Maybe 10px is fine?